src/main/java/com/wiley/autotest/selenium/elements/upgrade/v3/TeasyFluentWait.java

```diff
@@ -25,6 +26,7 @@ public TeasyFluentWait(T input, SearchStrategy strategy) {
         this.nullOnFailure = strategy.isNullOnFailure();
     }
 
+    @Nullable
```

Comment:
Do you really need this annotation here?

Reply:
check out what Nullable annotation is used for- this will probably answer your question.
in short - "yes, why not?"